test/LoadEnv/ParseSpec.hs

```diff
@@ -97,13 +97,16 @@ spec = do
                 `shouldBe` Right ("S3_KEY", "abc123")
             parse parseVariable "" "_S3_KEY=abc123\n"
                 `shouldBe` Right ("_S3_KEY", "abc123")
+            parse parseVariable "" "S3_key=abc123\n"
+                `shouldBe` Right ("S3_key", "abc123")
+            parse parseVariable "" "s3_key=abc123\n"
+                `shouldBe` Right ("s3_key", "abc123")
+
```

Comment:
Please remove this extra line of white space.

Reply:
:+1: